styles.css

```diff
@@ -0,0 +1,146 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Helvetica, sans-serif;
+}
+
+body {
+  text-align: center;
+}
+
+nav {
+  position: relative;
+  top: 100px;
+  width: 87vw;
+  height: 40px;
+  border-radius: 5px;
+  background: linear-gradient(180deg, #5f635f, lightgray, #5f635f);
+  margin-left: auto;
+  margin-right: auto;
+}
+
+nav ul li {
+  display: inline-block;
+}
+
+.nav_li {
+  text-align: center;
+  width: 100px;
+  color: white;
+  font-weight: 400;
+  font-size: 16px;
+  border-right: 1px solid grey;
+  padding: 10px;
+}
+
+nav li:hover {
+  cursor: pointer;
+}
+
+.nav_search {
+  border-radius: 10px;
+  background: lightgray url(assets/search.png) left no-repeat;
+  padding-left: 15px;
+  width: 100%;
+  height: 22px;
+  color: white;
+  margin-left: 3px;
+  margin-bottom: 10px;
+}
+
+.main_content {
+  background-image: url(assets/main.jpg);
+  background-repeat: no-repeat;
+  background-size: 100vw 940px;
+  width: 100vw;
+  height: 940px;
+  text-align: center;
+  color: #fff;
+}
+
+.main_content_section_a {
+  position: relative;
+  top: 200px;
+}
+
+.main_heading_a {
+  font-size: 73px;
+}
+
+.main_heading_b {
+  font-size: 50px;
+  font-weight: lighter;
+}
+
+.sub_heading {
+  font-size: 36px;
+}
+
+.main_content p:hover {
+  cursor: pointer;
+}
+
+.main_content_b {
+  display: grid;
+  grid-template-columns: 25vw 25vw 25vw 25vw;
+  height: auto;
+  margin: 2px;
+}
+
+.thumbnail {
+  height: auto;
+  width: 25vw;
+  cursor: pointer;
+}
+
+footer {
+  width: 100vw;
```

Comment:
* [ ] Change ```width: 100vw;``` to ```width: 100%;```
That is it with this the horizontal scroll should not show again